src/expose/Expose.ts

```diff
@@ -217,7 +220,7 @@ class Expose {
 		// command options (option that takes priority, like --version etc)
 		for (i = 0, ii = options.length; i < ii; i++) {
 			opt = options[i];
-			if (opt.command && ctx.hasOpt(opt.name, true)) {
+			if (opt.command && ctx.hasOpt(opt.name, true) && ctx.getOpt(opt.name)) {
```

Comment:
Since now `options` will include all the flags and their boolean values, we only want to call `executeCommand` if the option is true. For example, the `--help` flag has a command and now exists in the context, but since it is false in a command like `tsd query --info d3`, we don't want to execute the help command.